common/security-features/subresource/image.py

```diff
@@ -1,7 +1,12 @@
-import os, sys, array, math, StringIO
+import os, sys, array, math
 sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
 import subresource
 
+try:
```

Comment:
Should this not be

`from six.StringIO import StringIO` ?

Reply:
Yes, we can use six library here. TBH I'm not sure which way is better...

Reply:
It's always better not to re-implement stuff; using `six` means we have consistent logic for imports like this and reduces the chance of accidentally writing bugs.

Also, its one line rather than four :D

Reply:
Updated code. Thank you!